### DISCUSSION TITLE
How to bind filepond input with livewire

### DISCUSSION THREAD

**Discussion by AggelosGalatis** [livewire/livewire#5652] [1. Help] [answered]
Hello. I would like to upload some files with filepond in livewire. Can anyone help me with an example on how to do it?

**Comment by joshhanley** [comment, marked as answer]
@AggelosGalatis you will need to write a custom implementation to get FilePond and Livewire working together, and probably using Alpine to facilitate it. I recommend searching the forums and this repo as I'm sure someone else has already created one.
